package.json

```diff
@@ -91,13 +95,13 @@
     "swiper": "^8.3.2",
     "tailwind-merge": "^1.3.0",
     "tailwindcss": "^3.1.4",
-    "yaml": "^2.2.2",
-    "@slack/web-api": "^6.5.1"
```

Comment:
Why you have removed this dependency from the package file?

Reply:
Nope , I have not removed any dependency , I have just added  dependencies 'cypress' and 'eslint plugin for cypress '

Reply:
No issues, this dependency got listed above.

Reply:
What do you mean by "take all the changes from master branch " , to resolve merge conflicts ? I will resolve them right away